ro_help/hub/templates/ngo/base.html

```diff
@@ -46,7 +46,8 @@ <h1 class="donate-title">
                 {% endblocktrans %}
             </div>
         </div>
-        <div class="need-call2action"><a class="button is-success">{% trans "Donate" %}</a></div>
+        {% url 'ngo-helper-create' red_cross_context_data.ngo.pk red_cross_context_data.pk as base %}
```

Comment:
Let's rename `red_cross_context_data` into `red_cross_need` (since every variable is a context data variable).
Also, let's pass `red_cross` and `red_cross_need` separately since the need may be missing.

Reply:
if the need is missing, then the button will not do anything by default.

Reply:
If the need is missing the button should not be displayed.